src/window/stores/performance.js

```diff
@@ -33,29 +33,27 @@ import { handleNewData, attachToMobx } from '~/api/PerformancePanel';
 import { setUser, toggleRaygun } from '~/helpers/loggingApi';
 import PerformanceWindowApi from '../api/PerformanceWindow';
 
-global.config = {
-  settings: null
-};
+global.config = null;
 
 if (IS_ELECTRON) {
   const { ipcRenderer } = window.require('electron');
 
-  global.config.settings = ipcRenderer.sendSync('configQueried');
-  const { user, telemetryEnabled } = global.config.settings;
+  global.config = ipcRenderer.sendSync('configQueried');
+  const { user, telemetryEnabled } = global.config;
   setUser(user);
   toggleRaygun(telemetryEnabled, false);
 
   ipcRenderer.on('configChanged', (_event, changed) => {
     _.forEach(changed, (v, k) => {
-      _.set(global.config.settings, k, v.new);
+      _.set(global.config, k, v.new);
     });
 
     if (_.has(changed, 'user.id')) {
-      setUser(_.get(global.config.settings, 'user'));
+      setUser(_.get(global.config, 'user'));
     }
 
     if (_.has(changed, 'telemetryEnabled')) {
-      toggleRaygun(_.get(global.config.settings, 'telemetryEnabled'));
+      toggleRaygun(_.get(global.config, 'telemetryEnabled'));
```

Comment:
Might be better to have these strings like 'telemetryEnabled' and 'user' accessed as constants instead of using magic strings.

Reply:
Ya, definitely